services/UserService.js

```diff
@@ -22,4 +22,19 @@ module.exports = class UserService {
       return error;
     }
   }
+
+  static async updateUser(body, username) {
+    try {
+      const user = await User.update(body, {
+        where: {
+          username: {
+            [Op.match]: username,
+          },
```

Comment:
why are you doing sql query for updates?

Reply:
I just checked the official site, I think I should use the .set() method if updating partial fields or .save() to change the value entirely instead of using sql sql query and .update() method. I will change this tonight or tomorrow.